static/js/publisher/form.js

```diff
@@ -113,14 +114,15 @@ function initForm(config, initialState, errors) {
   let state = JSON.parse(JSON.stringify(initialState));
 
   const stateInput = document.createElement("input");
-  stateInput.type = "hidden";
```

Comment:
Was it removed for debugging?

<img width="763" alt="screen shot 2018-12-20 at 17 20 33" src="https://user-images.githubusercontent.com/83575/50296817-97749800-047b-11e9-9a66-948f5c947bca.png">

Reply:
ahh good catch, didn't mean to remove it!